addons/mail/static/src/utils/throttle/tests/throttle_tests.js

```diff
@@ -41,7 +41,7 @@ QUnit.test('single call', async function (assert) {
 
     let hasInvokedFunc = false;
     const throttledFunc = throttle(
-        this.env,
+        this.messaging,
```

Comment:
In preparation for future (no `this` here), get `messaging` from `start` in all tests.

Reply:
This test has been adapted. I think it's better to adapt the others in the PR related to` data/serverData `since it already implies some heavy diff in tests.

Reply:
Indeed, only adapt those that you introduce.